### ISSUE TITLE
Improve sentence structure model training time

### ISSUE THREAD

**Issue by csvance** [csvance/FTBot#45]
It looks like the dual LSTM architecture and hidden size may be complete overkill, and hence causing very slow / ineffective training. Going to test moving  changing the dim to 64 and removing one of the LSTM's entirely.

Also the number of epochs should be scaled by the amount of training data available.
